blockend/picocard.js

```diff
@@ -100,9 +106,11 @@ export function unpack (buffer, secret) {
       text = card.t
       break // plain yay!
     case 1: // Moved decryption to '/frontend/encryption.js'
+      // TODO: move back to avoid recursive nested encrypt glitch :snicker:
```

Comment:
`blockend/mod/encryption.js` , `import decrypt` and call here?

Reply:
Nah I'm silly.... i didn't think this through.. At first i was shocked why it was moved cause i thought i had solved the issue/that's how it used to work in alpha.

But when i started using the keypad PR I realized that the usecases had changed!
I'm gonna remove that comment for now because i don't know the crypto-flow for this new app. 
But goal is to never present user with the encrypted messages.